tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,26 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = ContentViewFileRepoTestCase.org
+        cv = make_content_view({u'organization-id': org['id']})
+        product_info = ContentViewFileRepoTestCase.product
+        repo = self._make_file_repository_upload_contents(
+            {u'content-type': 'file', u'organization-id': org['id'], u'product-id':
+                product_info['id']})
+        ContentView.add_repository({u'id': cv['id'], u'repository-id': repo['id'],
```

Comment:
use proper formatting and indentation
```
        ContentView.add_repository({
            'id': cv['id'], 
            'repository-id': repo['id'],
            'organization-id': org['id']
})
```

Please apply to all places

Reply:
Thanks, Same has been taken care.